mingw-w64-opencascade/PKGBUILD

```
@@ -137,17 +143,21 @@ build() {
     -D3RDPARTY_OPENVR_LIBRARY_openvr_api="${MINGW_PREFIX}/lib/libopenvr_api.dll.a" \
     ../occt-${_pkgver2}
 
-    ${MINGW_PREFIX}/bin/cmake --build .
+  ${MINGW_PREFIX}/bin/cmake --build .
 }
 
 package() {
   #Static Install
   cd "${srcdir}/build-${MSYSTEM}-static"
-  DESTDIR=${pkgdir} cmake --build . --target install
+  DESTDIR="${pkgdir}" ${MINGW_PREFIX}/bin/cmake --build . --target install
 
   #Shared Install
   cd "${srcdir}/build-${MSYSTEM}-shared"
-  DESTDIR=${pkgdir} cmake --build . --target install
+  DESTDIR="${pkgdir}" ${MINGW_PREFIX}/bin/cmake --build . --target install
+
+  install -Dm644 "${srcdir}"/occt-${_pkgver2}/LICENSE_LGPL_21.txt "${pkgdir}"${MINGW_PREFIX}/share/licenses/${_realname}/LICENSE
```

I think this quoting needs to be:
```suggestion
  install -Dm644 "${srcdir}/occt-${_pkgver2}/LICENSE_LGPL_21.txt" "${pkgdir}${MINGW_PREFIX}/share/licenses/${_realname}/LICENSE"
```